for forking practice

erussell10 opened the pull request:
I know the "puts" is wrong

sareg0 commented:
Hey @erussell10, if you make a change to your files, then add, commit and push them they will join the rest of your pull request here.

erussell10 commented:
I believe I have completed this. Essentially, what I did was: rename the file to erussell.rb (it was giving me a "file already exists message and I didn't want to override in case) and moved it to the ruby file. Because liz.rb was still showing up in my remote directly despite having moved it on my local directory, I deleted liz.rb on github. I think this is ready for accepting the pull request.

Question: how do I commit a "delete"? Example - when I did a git status, it was showing liz.rb as being deleted (even though it was moved and renamed) and I didn't see a way to "add and commit" that concept so that my files on github would reflect my local files for that specific change. I assume that's intentional; any thoughts are appreciated.

sareg0 commented:
hey @erussell10 sorry for the delay. You're all merged in now! Hooray!
